### PR TITLE
Less verbose bytecode comparison output in CI

### PR DESCRIPTION
**Pull request by cameel**
Depends on #11262.

### CODE REVIEW

File: scripts/bytecodecompare/prepare_report.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ for (const optimize of [false, true])
                         if ('metadata' in contractResults && cleanString(contractResults.metadata) !== undefined)
                             metadata = contractResults.metadata
 
-                        console.log(filename + ':' + contractName + ' ' + bytecode)
+                        console.log('BYTECODE COMPARE TEST ' + filename + ':' + contractName + ' ' + bytecode)
```
Comment (ekpyron) [inline scripts/bytecodecompare/prepare_report.js]: Right, I just wanted to ask whether you did something like this :-).

Comment (cameel) [reply scripts/bytecodecompare/prepare_report.js]: Yeah. Still waiting to see the results.

As for a successful run, you can see how it looks like here: https://app.circleci.com/pipelines/github/ethereum/solidity/15012/workflows/46282180-d244-4043-8474-d2b22f1c9394/jobs/684330/steps

Comment (cameel) [reply scripts/bytecodecompare/prepare_report.js]: Looks like the negative case works fine too: https://app.circleci.com/pipelines/github/ethereum/solidity/15019/workflows/488b11bd-b8f9-414b-8457-a1483f42fed4/jobs/684593/steps

**Review by ekpyron — DISMISSED** [review]
Now I missed the CI run that had the differing reports - but I trust you had a look at it and it looked good, so I'm approving!

**Comment by cameel** [comment]
@ekpyron I posted a link to it above: https://app.circleci.com/pipelines/github/ethereum/solidity/15019/workflows/488b11bd-b8f9-414b-8457-a1483f42fed4/jobs/684593/steps

**Comment by ekpyron** [comment]
> @ekpyron I posted a link to it above: https://app.circleci.com/pipelines/github/ethereum/solidity/15019/workflows/488b11bd-b8f9-414b-8457-a1483f42fed4/jobs/684593/steps

Ah, nice - I had not reloaded the github page! Looks great!

**Review by mijovic — APPROVED** [review]

**Comment by mijovic** [comment]
Looks good, non-determinism was solved in other PR. which is already merged so this looks good to go.

**Review by ekpyron — APPROVED** [review]

**Pull request merged**